### PR TITLE
Plug

### PR DESCRIPTION
**Pull request by mitchellhenke**
For https://github.com/getsentry/raven-elixir/issues/16

@jeregrine

---
This change is [Reviewable](https://reviewable.io/reviews/getsentry/raven-elixir/31)

### CODE REVIEW

File: lib/sentry.ex [hunk 1]
```diff
@@ -62,14 +62,9 @@ defmodule Sentry do
   def send_event(event = %Event{}) do
     included_environments = Application.fetch_env!(:sentry, :included_environments)
     if Application.fetch_env!(:sentry, :environment_name) in included_environments do
```
Comment (jeregrine) [inline lib/sentry.ex]: Is this an oops?

Comment (jeregrine) [inline lib/sentry.ex]: Lets make an issue to fix this

File: lib/sentry/plug.ex [hunk 2]
```diff
@@ -0,0 +1,41 @@
+defmodule Sentry.Plug do
```
Comment (jeregrine) [inline lib/sentry/plug.ex]: Lets add some empty module docs

Comment (mitchellhenke) [inline lib/sentry/plug.ex]: added a short doc

**Comment by jeregrine** [comment]
LGTM

**Pull request merged**